src/helpers/approvals-satisfied.ts

```diff
@@ -30,13 +31,10 @@ export class ApprovalsSatisfied extends HelperInputs {
   pull_number?: string;
 }
 
-export const approvalsSatisfied = async ({
-  teams,
-  users,
-  number_of_reviewers = '1',
-  required_review_overrides,
-  pull_number
-}: ApprovalsSatisfied = {}) => {
+export const approvalsSatisfied = async (
+  { teams, users, number_of_reviewers = '1', required_review_overrides, pull_number }: ApprovalsSatisfied = {},
+  approvalsNotMetMessage: string | undefined = undefined
```

Comment:
Can we use the existing `body` input instead of this second argument for providing an optional comment body? `approvalsSatisfied` is a standalone action